### PR TITLE
data: Alexxanderr20

### PR DESCRIPTION
**Pull request by Alexxanderr20**
added data: Alexxanderr20

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7919"></a>

### CODE REVIEW

File: data/Alexxanderr20.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Edward Munoz" 
+  "bio": "Aspiring software engineer who is focused on a front-end tech stack, but is open to any kind of     software stack to learn and grow." ,
+  "links" : [
+   {
+    "name": "Follow me on LinkedIn" ,
+    "url": "https://www.linkedin.com/in/edward-alex-munoz/",
+    "icon": "FaLinkedIn"
```
Comment (Sindhuinti) [inline data/Alexxanderr20.json:8]: Please change your icon name to `FaLinkedin`.
docs:  https://linkfree.io/icons

**Review by Sindhuinti — CHANGES_REQUESTED** [review]

File: data/Alexxanderr20.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Edward Munoz" 
+  "bio": "Aspiring software engineer who is focused on a front-end tech stack, but is open to any kind of     software stack to learn and grow." ,
```
Comment (RohanSasne) [inline data/Alexxanderr20.json:3]:
```suggestion
  "bio": "Aspiring software engineer who is focused on a front-end tech stack, but is open to any kind of software stack to learn and grow.",
```

extra white spaces can be removed for consistency throughout the repository :)

File: data/Alexxanderr20.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  "links" : [
+   {
+    "name": "Follow me on LinkedIn" ,
```
Comment (RohanSasne) [inline data/Alexxanderr20.json:6]:
```suggestion
    "name": "Follow me on LinkedIn",
```

Same follows over here

**Review by RohanSasne — CHANGES_REQUESTED** [review]

File: data/Alexxanderr20.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Edward Munoz"
```
Comment (kumarsonsoff3) [inline data/Alexxanderr20.json:2]: Please add the missing comma here to validate your json! 

```suggestion
  "name": "Edward Munoz",
```

File: data/Alexxanderr20.json (hunk 1 above)
Comment (kumarsonsoff3) [reply data/Alexxanderr20.json:8]:
```suggestion
    "icon": "FaLinkedin"
```

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good. Great collaboration everyone. 👍

**Review by RohanSasne — APPROVED** [review]
lgtm

**Pull request merged**